tagcompare/test/test_image.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import os
+from tagcompare import image
+from tagcompare import settings
+
+settings.TEST_MODE = True
```

Comment:
I don't think you need this.  I think this flag is just set for cases where the test needs to use the flag.  I'm honestly not sure I thought that through when I first used it so you shouldn't use it if you don't need to :)